src/main/java/io/github/dsheirer/record/wave/AudioMetadataUtils.java

```diff
@@ -89,38 +89,35 @@ public static Map<AudioMetadata, String> getMetadataMap(IdentifierCollection ide
 
         if(identifierCollection != null)
         {
-            for(Identifier to: identifierCollection.getIdentifiers(Role.TO))
-            {
-                StringBuilder sb = new StringBuilder();
-                sb.append(to.toString());
+            StringBuilder sb;
 
-                List<Alias> aliases = aliasList.getAliases(to);
+            Identifier to = identifierCollection.getToIdentifier();
+            sb = new StringBuilder();
+            sb.append(to.toString());
```

Comment:
There's potential for 'to' identifier to be null here.  Please add null check/safety.

Previously, the for/each loop protected against a null identifier because it would only loop on the non-null returned values.